context/AuthContextProvider.tsx

```diff
@@ -54,6 +56,14 @@ export const AuthContextProvider = ({
     setIsSignedIn(true);
   }, [user]);
 
+  useMemo(() => {
+    if (isSignedIn) {
```

Comment:
Have you been able to test if this works as intended for users with the Super Admin and the Admin roles? And have you been able to test if the regular users are able to access the regular content as intended?

Reply:
Yup! I was able to test for both scenarios.